### ISSUE TITLE
Cannot use this module from TypeScript

### ISSUE THREAD

**Issue by OnkelTem** [TooTallNate/node-speaker#159]
When I build my package which uses `node-speaker` I get this:

```
node_modules/speaker/index.d.ts:3:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.

3 namespace Speaker {
  ~~~~~~~~~
```

I don't really get how to fix this issue, have never seen it before.

Any ideas?

**Comment by tjjfvi** [comment]
Fixed by #153

**Comment by LinusU** [comment]
Fixed published as [0.5.3](https://github.com/TooTallNate/node-speaker/releases/tag/v0.5.3)
